### PR TITLE
merge Rating comp

### PR DESCRIPTION
**Pull request by piyushsh25**
Screenshot (205)

### CODE REVIEW

File: components/modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="modal.css">
+    <title>Document</title>
```
Comment (junaidshaikh-js) [inline components/modal/modal.html:9]: title name should be meaningful

**Review by junaidshaikh-js — COMMENTED** [review]
Looks like you also pushed modal component with ratings.

File: components/modal/modal.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+.confirmation-modal-action-action-2 {
+    padding: var(--size5);
+}
+
+@media only screen and (max-width:830px) {
+    .simple-modal-action-action-1 {
+        margin-right: var(--size6);
+        margin-top: var(--size0);
+    }
+}
```
Comment (hinanshisuthar) [inline components/modal/modal.css:134]: Add a new line at the EOF.

File: components/modal/modal.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                </div>
+                <div class="simple-modal-action-action-2">
+                    Action-2
+                </div>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```
Comment (hinanshisuthar) [inline components/modal/modal.html:91]: Here, too.

Comment (piyushsh25) [reply components/modal/modal.html:91]: the code formatter extension isn't allowing me to add a new line at the end.

**Pull request merged**